lib/server.js

```diff
@@ -25,6 +25,13 @@ module.exports = function (gulp, gutil) {
     force: false,
     pluginPoints: [],
     sdkOutputDir: undefined,
+    useLocalCompile: false,
+    localSkinCompileVersion: 'v1.6-lia16.1',
```

Comment:
I would like lithium-sdk to be independent of lia release. Can we move these properties to sdk.conf.json in angular-li.
Difference between sdk.conf.json and server.conf.json is that sdk.conf.json is committed to github where as server.conf.json is ignored. sdk.conf.json is generic where as server.conf.json is local to the user and community.